ext/mysqli/mysqli.stub.php

```diff
@@ -623,14 +587,11 @@ public function get_result() {}
 
 final class mysqli_warning
```

Comment:
I don't see how these properties can be null? Is it because there is no assert? The values are populated at the time of the creation of the object.

Reply:
I'm not exactly sure, but I think they are nullable because there are no `CHECK_STATUS` calls in the property handlers (i.e. `mysqli_warning_message`)

Reply:
That's because `mysqli_warning` doesn't have status. It is always ready. You can't instantiate it on its own. The values come from `SHOW WARNINGS` and `sqlstate` is always hardcoded. I see no chance that any of them can ever be null. See function `php_new_warning`.

Reply:
ah, I see! Thanks for the explanation!

Reply:
I've just got rid of the nullable types